### DISCUSSION TITLE
.net 6?

### DISCUSSION THREAD

**Discussion by troglobytor** [Zeugma440/atldotnet#135] [Ideas]
I did not see .net 6 support. Is this something that can be down with source?

**Comment by Zeugma440** [comment]
I'm not sure I understand what your goal is.

If you need to run the library on a .NET 6 environment, it will definitely work as it is compatible with previous versions.

If you want to modify the code using .NET 6 features (such as C# 10), you're welcome to create your own fork. ATL aims at staying compatible with multiple environments and platforms, and won't jump to being .NET 6-only anytime soon.

**Comment by troglobytor** [comment]
I had read "The ATL library runs on .NET Core 3.1+ / .NET Standard 2.1+ / .NET Framework 4.8+" and thought there might be something, though unlikely, with .NET Core 6. Thanks for answer.

**Comment by sandreas** [comment]
If you would like to see a .NET 6 *reference implementation*, you could take a look at [`tone`](https://github.com/sandreas/tone)... Completely .NET 6 using atldotnet `4.0.8` (nuget `4.8.0`)
